lib/3_add_up_numbers.js

```diff
@@ -6,6 +6,10 @@
 
 module.exports = {
   add_up_numbers: function(arr){
-    // Your code goes here.
+  var sum=0;
+for (var i=0; i<arr.length; i++){
```

Comment:
Indentation also an issue here. Try indenting similarly to the guidelines I gave for problem 4.